app/server/lib/SQLiteDB.ts

```diff
@@ -365,6 +375,7 @@ export class SQLiteDB implements ISQLiteDB {
     const alreadyClosed = this._closed;
     this._closed = true;
     if (!alreadyClosed) {
+      this._pauseReject?.();
```

Comment:
This would cause `await this._applyPause()` to throw `undefined` as exception. I think specifying an Error object would be much better.

Reply:
Added an exception.